### PR TITLE
fix: adjust z-indices to make widget actions available

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
[MUMUP-3345](https://jira.doit.wisc.edu/jira/browse/MUMUP-3345): "Allow users to remove widgets that have an overlay"

**In this PR**:
- Widget remove button has higher priority than widget overlays
- Top-down adjustment of z-indices throughout app CSS (with LESS variables)
    - Dialogs/toasts always on top
    - Top bar higher than content
    - Menus higher than the menu focus trap
    - Widget actions higher than widget overlay
- Removed a couple unused/unnecessary z-index statements

----

PR considerations checklist:

- [x] Updates `CHANGELOG.md` to reflect this PR's change.
- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,16 +15,18 @@ Versioning](http://semver.org/spec/v2.0.0.html).
  
 ### Changed
 * "About" page now sources meaningful content from `aboutPageURL` (#755)
+* z-index values adjusted from highest (101) to lowest (51) stated values to play nicer with Angular Material (#760)
```
Comment (apetro) [inline CHANGELOG.md]: How impactful is this change? Is this going to feel like a breaking change to upgrading framework apps? What will app developers typically need to do in updating apps to a framework version with this change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,16 +15,18 @@ Versioning](http://semver.org/spec/v2.0.0.html).
...
 * When priority notification title is truncated, provide full title as tooltip
   (#754)
 * Fix route to newly-added version info page (#758)
-
+* Widgets with overlays can be removed (#760)
+ 
 ### Removed
 
-* no loger used LoginOnLoad option removed (#753)
+* No logger used LoginOnLoad option removed (#753)
```
Comment (apetro) [inline CHANGELOG.md]: Better, but still not quite right. :)

Comment (vertein) [inline CHANGELOG.md]: longer*

Comment (thevoiceofzeke) [reply CHANGELOG.md]: Oops...

File: CHANGELOG.md (hunk 1 above)
Comment (thevoiceofzeke) [reply CHANGELOG.md]: It shouldn't make much of an impact. I would guess most downstream devs would be writing custom z-indices to make content sit on top of our top bar or other high-index content, in which case they would be unaffected. The only case in which they might be affected is if they have z-indices for content sitting between the top bar and some other content (like the menu), which seems unlikely.

**Pull request merged**